distributed/deploy/tests/test_adaptive.py

```diff
@@ -460,3 +424,58 @@ async def test_update_adaptive(cleanup):
         await asyncio.sleep(0.2)
         assert first.periodic_callback is None
         assert second.periodic_callback.is_running()
+
+
+from distributed.utils_test import slowinc
```

Comment:
This appears to be already imported at the top of this module

Reply:
Wondering why me pre-commit didn't catch that... will fix of course